cmd/deploy/deploy.go

```diff
@@ -202,6 +206,7 @@ func Deploy(ctx context.Context) *cobra.Command {
 				K8sClientProvider:  okteto.NewK8sClientProvider(),
 				Builder:            buildv2.NewBuilderFromScratch(),
 				GetExternalControl: GetExternalControl,
+				Fs:                 afero.NewOsFs(),
```

Comment:
We should be adding this into all deployCommand definition.

Maybe creating a constructor is the best way to go if we don't want to have unwanted panics

Reply:
I think we are adding panics into init and up commands

Reply:
I will keep an eye on it, it is true that it is prone to panics